test/ref/pointwise.cpp

```diff
@@ -49,3 +52,28 @@ TEST_CASE(pointwise_test)
     std::vector<float> gold = {0, 2, 4};
     EXPECT(migraphx::verify::verify_rms_range(results_vector, gold));
 }
+
+TEST_CASE(pointwise_multi_out_test)
```

Comment:
Would current logic handle case of nested tuples ? 
If so can you add a test for the same ?

Reply:
Right now there isnt a way to create a tuple in the pointwise module.

Reply:
> Right now there isnt a way to create a tuple in the pointwise module. 

I assume you mean there is no way to create "nested" tuples as the outputs of the pointwise module.  Tuples can be created that is what this PR is doing.

Reply:
> Tuples can be created that is what this PR is doing.

Sure in general yes, but they can't be created in a pointwise module.